src/Entity/Click.php

```diff
@@ -1,158 +1,97 @@
 <?php
 
-namespace AppBundle\Entity;
+namespace App\Entity;
 
 use Doctrine\ORM\Mapping as ORM;
 
 /**
  * Click
  *
  * @ORM\Table(name="click")
- * @ORM\Entity(repositoryClass="AppBundle\Repository\ClickRepository")
+ * @ORM\Entity(repositoryClass="App\Repository\ClickRepository")
  */
 class Click
 {
     /**
-     * @var int
      *
      * @ORM\Column(name="id", type="integer")
      * @ORM\Id
      * @ORM\GeneratedValue(strategy="AUTO")
      */
-    private $id;
+    private int $id;
```

Comment:
```suggestion
    private ?int $id = null;
```
This is a general mistake for typed properties.
When a class has `private int $id;` without explicit property initialization, after creating an object and accessing the property, PHP throws an exception
```php
class Demo
{
    private int $id;

    public function getId(): int
    {
        return $this->id;
    }
}

$obj = new Demo();
echo $obj->getId();
```
```
PHP Fatal error:  Uncaught Error: Typed property Demo::$id must not be accessed before initialization in scratches/scratch_1.php:9
Stack trace:
#0 scratches/scratch_1.php(14): Demo->getId()
#1 {main}
  thrown in scratches/scratch_1.php on line 9

Process finished with exit code 255
```